pkg/ebpf/driver_interface.go

```diff
@@ -142,18 +153,27 @@ func (dc *DriverInterface) prepareDriverFilters() error {
 // To capture all traffic for an interface, we create an inbound/outbound traffic filter
 // for both IPV4 and IPV6 traffic going to that interface
 func createFiltersForInterface(iface net.Interface) (filters []C.struct__filterDefinition) {
-	filters = append(filters, newDDAPIFilter(C.DIRECTION_INBOUND, iface.Index, true))
-	filters = append(filters, newDDAPIFilter(C.DIRECTION_OUTBOUND, iface.Index, true))
-	filters = append(filters, newDDAPIFilter(C.DIRECTION_INBOUND, iface.Index, false))
-	filters = append(filters, newDDAPIFilter(C.DIRECTION_OUTBOUND, iface.Index, false))
+	filters = append(filters, newDDAPIFilter(C.DIRECTION_OUTBOUND, C.FILTER_LAYER_ALE_CONNECT, iface.Index, true))
+	filters = append(filters, newDDAPIFilter(C.DIRECTION_OUTBOUND, C.FILTER_LAYER_TRANSPORT, iface.Index, true))
+
+	filters = append(filters, newDDAPIFilter(C.DIRECTION_INBOUND, C.FILTER_LAYER_ALE_RECVCONN, iface.Index, true))
+	filters = append(filters, newDDAPIFilter(C.DIRECTION_INBOUND, C.FILTER_LAYER_TRANSPORT, iface.Index, true))
+
+	// this one doesn't really have a "direction".  TODO  Should probably clarify the API
+	filters = append(filters, newDDAPIFilter(C.DIRECTION_INBOUND, C.FILTER_LAYER_ALE_CLOSURE, iface.Index, true))
+
+	// for now skip ipv6
```

Comment:
Is there a reason we're skipping ipv6 for now?

Reply:
I was being lazy :)